components/messages/HowTo.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from "./HowTo.module.css";
+import Button from "../button/Button";
+
+function HowTo() {
+  return (
+    <div className={styles.howtocontainer}>
+      <span className={styles.title}>How it works</span>
```

Comment:
This could be a `h` element instead of a `span`. Changing this would make it easier to target in CSS: `.howtocontainer h1` ie.
Would save you some classes :)